addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_nodes.py

```diff
@@ -95,6 +95,8 @@ def __gather_children(blender_object, export_settings):
     children = []
     # standard children
     for child_object in blender_object.children:
+        if child_object.parent_bone:
+            continue
```

Comment:
Huh, so these objects are both parented to the armature and the bone, in blender's representation? That's surprising, a comment in the code here might be helpful.

Reply:
Yeah it's a bit confusing ... it's because in blender bones are apparently not objects like everything else, but special 'sub objects' of the armature object